micro_dl/plotting/plot_utils.py

```diff
@@ -74,6 +76,98 @@ def save_predicted_images(input_batch, target_batch, pred_batch,
         plt.close(fig)
 
 
+def plot_xyz(image_dir,
```

Comment:
plot_xyz sounds like a scatter plot in 3D, could we please rename this to: save_center_orthogonal_view
or save_mid_orthogonal_slice or the like?